core/controllers/editor.py

```diff
@@ -492,6 +436,56 @@ def put(self, exploration_id):
         })
 
 
+class ExplorationStatusHandler(EditorHandler):
+    """Handles publishing of an exploration."""
+
+    def _publish_exploration(self, exploration_id, make_public):
```

Comment:
Document these, using the usual Args, Raises etc. format. I know they're private methods, but I think it's still worth documenting them because it's not obvious that make_public can be False.